rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -613,11 +609,11 @@ public String getMyPublicIpFromRemoteService(){
                 throw new IOException("Invalid address: '" + externalIp + "'");
             }
 
-            tryParseIpOrThrow();
+            tryParseIpOrThrow(externalIp);
             logger.info("External address identified: {}", externalIp);
         } catch (IOException e) {
             logger.error("Can't get external IP. " + e);
-            externalIp = getPeerDiscoveryBindAddress();
+            externalIp = getBindAddress().toString();
```

Comment:
0.0.0.0 is a valid bind address but is not a valid external IP.

Reply:
:+1: I will fix this, the current implementation would throw an exception if no valid external could be found anyway